Gems/Atom/Feature/Common/Code/Source/CoreLights/SimplePointLightFeatureProcessor.cpp

```diff
@@ -11,12 +11,17 @@
 
 #include <AzCore/Math/Vector3.h>
 #include <AzCore/Math/Color.h>
+#include <Atom/Feature/CoreLights/LightCommon.h>
 #include <Atom/Feature/Mesh/MeshFeatureProcessor.h>
 #include <Atom/RHI/Factory.h>
 #include <Atom/RPI.Public/ColorManagement/TransformColor.h>
+#include <Atom/RPI.Public/RenderPipeline.h>
 #include <Atom/RPI.Public/RPISystemInterface.h>
 #include <Atom/RPI.Public/Scene.h>
 #include <Atom/RPI.Public/View.h>
+#include <numeric>
+
+AZ_CVAR(int, r_maxVisiblePointLights, -1, nullptr, AZ::ConsoleFunctorFlags::DontReplicate, "Maximum number of visible point lights to use when culling is not available. -1 means no limit");
```

Comment:
I would set to 20 so it would be consistent with the shader define. You can add the shader's file link here.

Reply:
The only issue with that is that 20 is just the number for non-mobile pipelines that do not do gpu culling. It is actually 5 for mobile pipeline. Having the default as -1 feels better in this case. What are your thoughts @akioCL?

Reply:
Maybe add some comment to explain the use cases. Also, mention that the vcar is only valid for that context in the cvar's description.